src/main/java/de/uhd/ifi/se/decision/management/jira/changeimpactanalysis/ChangePropagationRule.java

```diff
@@ -14,10 +20,22 @@
 public enum ChangePropagationRule {
 
 	STOP_AT_SAME_ELEMENT_TYPE("Stop at elements with the same type as the selected element",
-			new StopAtSameElementType()), //
-	IGNORE_INCOMING_LINKS("Outward links only", new IgnoreIncomingLinks()), //
+			new StopAtSameElementType()),
+	IGNORE_INCOMING_LINKS("Outward links only", new IgnoreIncomingLinks()),
 	BOOST_WHEN_TEXTUAL_SIMILAR("Boost when element is textual similar to the selected element",
-			new BoostWhenTextualSimilar());
+			new BoostWhenTextualSimilar()),
+	BOOST_WHEN_HIGH_AMOUNT_OF_DISTINCT_AUTHORS("Boost when element has a large number of distinct update authors",
+			new BoostWhenHighAmountOfDistinctAuthors()),
+	BOOST_WHEN_EQUAL_COMPONENT("Boost when element is assigned the same component",
+			new BoostWhenEqualComponent()),
+	BOOST_WHEN_EQUAL_DECISION_GROUP("Boost when element is assigned the same decision group",
+			new BoostWhenEqualDecisionGroup()),
+	BOOST_WHEN_MORE_OUTBOUND_THAN_INBOUND("Boost when element has more outbound than inbound links",
+			new BoostWhenMoreOutboundLinks()),
+	BOOST_WHEN_LOW_AVERAGE_AGE("Boost when element has a low average age",
+			new BoostWhenLowAverageAge()),
+	BOOST_WHEN_COUPLED("Boost when element is coupled to the selected element",

```

Comment:
I suggest to rename this class "BoostWhenEvolutionaryCoupled" to be more verbose.